apps/db_data/models.py

```diff
@@ -179,13 +179,24 @@ class Event(models.Model):
         on_delete=models.PROTECT,
         help_text="Currently unused.",
     )
-
-    def __str__(self):
-        return f"{self.name} ({self.date})"
```

Comment:
I made a change to that to be consistent with the DateTimeField. I made a null check because of an issue that came up after migrating and all events had null `date_time`, which messed up the admin interface.
```py
def __str__(self):
    if not self.date_time:
        return f"{self.name}"
    return f"{self.name} ({self.get_date_string()})"
```

Reply:
Maybe we can keep the old `date` and `time` fields for backward compatibility?

Reply:
We could configure the migration to convert from the existing date field to the new datetime field

Reply:
Just the date right, because the current CharField for time is harder to parse because there isn't a consistent pattern

Reply:
That's right, the CharField was just there as a hack to display a time range. You can just ignore it, since those are only events in the past.